From: Danila Tikhonov <danila@jiaxyga.com>
Date: Tue, 22 Apr 2025 20:13:46 +0300
Subject: FROMLIST: remoteproc: qcom: pas: Add SM7150 remoteproc support

Add DSP Peripheral Authentication Service support for the SM7150
platform.

Signed-off-by: Danila Tikhonov <danila@jiaxyga.com>
---
 drivers/remoteproc/qcom_q6v5_pas.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/remoteproc/qcom_q6v5_pas.c b/drivers/remoteproc/qcom_q6v5_pas.c
index 158bcd6..302a09f 100644
--- a/drivers/remoteproc/qcom_q6v5_pas.c
+++ b/drivers/remoteproc/qcom_q6v5_pas.c
@@ -1502,6 +1502,9 @@ static const struct of_device_id qcom_pas_of_match[] = {
 	{ .compatible = "qcom,sm6375-adsp-pas", .data = &sm6350_adsp_resource},
 	{ .compatible = "qcom,sm6375-cdsp-pas", .data = &sm8150_cdsp_resource},
 	{ .compatible = "qcom,sm6375-mpss-pas", .data = &sm6375_mpss_resource},
+	{ .compatible = "qcom,sm7150-adsp-pas", .data = &sm8350_adsp_resource},
+	{ .compatible = "qcom,sm7150-cdsp-pas", .data = &sm6350_cdsp_resource},
+	{ .compatible = "qcom,sm7150-mpss-pas", .data = &mpss_resource_init},
 	{ .compatible = "qcom,sm8150-adsp-pas", .data = &sm8150_adsp_resource},
 	{ .compatible = "qcom,sm8150-cdsp-pas", .data = &sm8150_cdsp_resource},
 	{ .compatible = "qcom,sm8150-mpss-pas", .data = &mpss_resource_init},
